content/cpp/concepts/data-types/data-types.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Data Types'
-Description: 'C++ provides a rather large number of types. However, you can write perfectly good programs using only five of those: int is a type for storing integer (whole) numbers. An integer typically requires 4 bytes of memory space and ranges from -2³¹ to 2³¹. cpp int year = 1991; int age = 28;  double is a type for storing floating point (decimal) numbers. Double variables typically require 8 bytes of memory space.'
+Description: 'C++ supports many **data types** that represent the size and kind of values being stored in memory. You can write perfectly good programs using only five of those.'
```

Comment:
```suggestion
Description: 'C++ supports many data type that represent the size and kind of values being stored in memory.'
```

No need for emphasis in the metadata. Also, let's stick with a neutral, manual voice instead of a "you" or "we" voice.